src/doc/brush.cpp

```diff
@@ -312,13 +316,142 @@ void Brush::clean()
   m_image.reset();
   m_maskBitmap.reset();
   m_backupImage.reset();
+  resetSymmetries();
 }
 
 static void algo_hline(int x1, int y, int x2, void *data)
 {
   draw_hline(reinterpret_cast<Image*>(data), x1, y, x2, BitmapTraits::max_value);
 }
 
+void Brush::resetSymmetries()
```

Comment:
Something I don't quite like is the fact that these vectors are always resized in `resetSymmetries()`, even when they are never used. I think `resetSymmetries()` could do nothing if `m_symmetryImages` is empty, and we could add a new private member function `reserveSymmetries()` that does what `resetSymmetries` does now and that is called from `getSymmetryImage/Mask` when needed.